UKHomeOffice/nmsw-ui
# NMSW-1125 Create search bar component

# Ticket

NMSW-1125

----

## AC

Build a search bar component - no search functionality

----

## To test

run app
sign in
click on a declaration
upload passengers
on upt page
-> you should see a search bar component
-> Type something into it
click submit
-> You should see the data logged in the console


----

## Notes
